Logging/ARKLogMessage.h

```diff
@@ -18,8 +18,9 @@
 //  limitations under the License.
 //
 
-#import <Aardvark/Aardvark.h>
+#import "ARKLogType.h"
 
+@import UIKit;
```

Comment:
I'd prefer to `#import <UIKit/UIKit.h>` for now. We'll move to `@import` syntax all at once.